dali/pipeline/operators/crop/slice.cc

```diff
@@ -18,70 +18,42 @@ namespace dali {
 
 DALI_SCHEMA(Slice)
     .DocStr(
-        R"code(Crop a slice of a defined `size` from an `input` tensor, staring
-at the location specified by `begin`. Inputs must be supplied as 3 Tensors in a
-specific order: `Images` containing image data in NHWC format, `Begin` containing
-the starting pixel coordinates for the `crop` in `(x,y)` format, and `Size` containing
-the pixel dimensions of the `crop` in `(w,h)` format. For both `Begin` and `Size`,
-coordinates must be in the interval `[0.0, 1.0]`. The resulting tensor output of
-Slice operation is a cropped version of the input tensor `Images`.)code")
+        R"code(Extract a subtensor or `slice` with a given shape and anchor.
```

Comment:
@JanuszL do you agree with the last statement?

Reply:
So we need to rework operator we use a source of that input for slice and update the examples, right?

Reply:
Correct. For now, we can live with both formats but I believe we should unify it eventually